Makefile

```
@@ -7,7 +7,10 @@ IMAGE_TAG ?= $(GIT_BRANCH)-$(GIT_REVISION)
 GOOS ?= $(shell go env GOOS)
 GOARCH ?= $(shell go env GOARCH)
 
-rollout-operator:
+DONT_FIND := -name vendor -prune -o -name .git -prune -o -name .cache -prune -o -name .pkg -prune -o
+GO_FILES := $(shell find . $(DONT_FIND) -name cmd -prune -o -name '*.pb.go' -prune -o -type f -name '*.go' -print)
```

```suggestion
GO_FILES := $(shell find . $(DONT_FIND) -type f -name '*.go' -print)
```
I think these prunes can be removed.

As I understand it the intent of `-name cmd -prune -o` is to ignore irrelevant code when there are multiple directories in `cmd` but something else should include the directory that matters, but the latter half of that is missing here so modifying `cmd/rollout-operator/main.go` doesn't trigger a rebuild. There's only `cmd/rollout-operator` here anyway so it seems less confusing to leave it out.

There are currently no `'*.pb.go'` files outside of `vendor`, which is already pruned.

---

Thank you!